test/resources/workflow-runs-outputs.test.ts

```diff
@@ -0,0 +1,138 @@
+import { faker } from "@faker-js/faker";
+import { Applicant, WorkflowRun } from "onfido-node";
+
+import { exampleWorkflowRun } from "../test-examples";
+import {
+  cleanUpApplicants,
+  completeTask,
+  createApplicant,
+  createWorkflowRun,
+  getExpectedObject,
+  onfido,
+  uploadDocument,
+  uploadLivePhoto
+} from "../test-helpers";
+
+function getExpectedWorkflowRun(
+  exampleWorkflowRun: WorkflowRun,
+  overrideProperties = {}
+) {
+  return getExpectedObject(exampleWorkflowRun, {
+    applicant_id: expect.stringMatching(/^[0-9a-z-]+$/),
+    id: expect.stringMatching(/^[0-9a-z-]+$/),
+    workflow_id: expect.stringMatching(/^[0-9a-z-]+$/),
+    workflow_version_id: expect.anything(),
+    dashboard_url: expect.anything(),
+    status: expect.anything(),
+    output: expect.anything(),
```

Comment:
I think it would be worth it validating `output` field to be sure it's accessible even if not defined in detail.

Reply:
I ran these tests locally, and they're returning `null` in the output. Is that expected?

Reply:
We haven't validated enough this output field yet, but the goal of improving these tests over was also to be sure that what is returned in output field, even if can't be mapped to a precise schema, it can be accessible.

Therefore probably workflow run should be improved to have some data in this field so that we can see such data can be accessed using client library.